source/includes/getting-started/_concepts.html.md

```diff
@@ -1,61 +1,67 @@
-* **Data:** is a training source. It is information related to the concepts being trained that are used for training or test and evaluate how well the training is implemented. Non-generated data may sometimes be difficult to find, and the data that exists may be a very small set or difficult to label. An example of a data set you could train with is the MNIST database, which is a collection of handwritten digits used for training image processing systems and other machine learning systems.
+* **Data:** is also a training source. It is information related to the scenario being trained, or to test and evaluate how well the training is implemented. Non-generated data may sometimes be difficult to find, and the data that exists may be a very small set or difficult to label. An example of a data set you could train with is the [MNIST database][5], which is a collection of handwritten digits used for training image processing systems and other machine learning systems.
+
+**Note:** Currently, during our private beta, you can only use simulators as your training source.
 
-**Note:** Currently, during our private beta, you can only train with simulators as your training source.
+[//]: # (Simulators are the only training source, beta.bons.ai is current website instead of brains.bons.ai)
 
-## Teach
+###### Teach
 
 * **Inkling:** our unique, special purpose, pedagogical programming language. Pedagogical programming is a new programming paradigm. It expresses the solution to a problem in terms of how you teach the computer to find the solution as opposed to how to calculate the solution.
-* **Bonsai Artificial Intelligence Engine:** abstracts away and automates the low-level mechanics of artificial intelligence. Each Inkling program developed is fed into the Bonsai Artificial Intelligence Engine in order to generate and train appropriate models (referred to as BRAINs).
+* **Bonsai Artificial Intelligence (AI) Engine:** abstracts away and automates the low-level mechanics of artificial intelligence. Each Inkling program developed is fed into the Bonsai AI Engine in order to generate and train appropriate models (referred to as BRAINs).
 
-## Use
+###### Use
 
 * **Mastermind:** the Bonsai AI application for building AI. This tool will contain our Integrated Development Environment (IDE). **Note:** This tool will not be available for the private beta.
-* **SDK & Libraries:** enables you to connect your own simulators to the Bonsai Artificial Intelligence Engine. It also enables you to integrate your application with your BRAIN
-* **HTTP API:** enables AI creators and AI users to interact with the Bonsai Artificial Intelligence Engine.
+* **SDK & Libraries:** enables you to connect your own simulators to the Bonsai AI Engine. They also enable you to integrate your application with your BRAIN.
+* **HTTP API:** enables AI creators and AI users to interact with the Bonsai AI Engine.
```

Comment:
Alright I've made a task for it in Maniphest, it should probably be part of the larger overhaul.